src/qt/walletview.cpp

```diff
@@ -197,6 +200,81 @@ void WalletView::gotoVerifyMessageTab(QString addr)
         signVerifyMessageDialog->setAddress_VM(addr);
 }
 
+void WalletView::gotoLoadPSBT()
+{
+    QString filename = GUIUtil::getOpenFileName(this,
+        tr("Load Transaction Data"), QString(),
+        tr("Partially Signed Transaction (*.psbt)"), nullptr);
+    if (filename.isEmpty()) return;
+    const std::streamsize max_file_size = 100000000;
```

Comment:
nit: I feel like this should be some sort of filesize constant somewhere

Reply:
A UTXO snapshot can be multiple gigabytes, but a wallet or PSBT is expected to be much smaller. So I don't think there's a generic value. PSBT also doesn't specify a maximum size, so `PSBT_MAX_FILE_SIZE` could give the wrong impression (nothing a comment can't solve).